### DISCUSSION TITLE
Feedback from View Assist users

### DISCUSSION THREAD

**Discussion by dinki** [dinki/View-Assist#4] [General]
Hey all.  I'm humbled by the nice things that has been shared about the project.  I know that some have tried VA and I'm curious to hear your thoughts and any feedback that can make the project better.  I have some major updates coming but would also like to hear your current impressions as I work towards providing the instructions for the upgrade.  Thanks!

**Comment by MrTango** [comment]
Hi @dinki thx again for your work. I have StreamAssist now working with RtpMic v1.13 manually installed.
I can turn on a light by saying "turn on the office light", but i can't turn it on by saying just "turn on the light".
Is it possible that the location of the tablet is not set to office?
I set area office for the browser mode client which I am using for the tablet.
Also i set area office for the StreamAssist integration.
But i get an no_valid_targets in the debug of that assistant if i don't state the room.
Any idea if that can be solved?
The S3 Box does this really well.

**Comment by dinki** [comment]
Hi there!  To follow up on your other post, I am using the version of RtpMic from the Playstore:

https://play.google.com/store/apps/details?id=com.rtpmic&hl=en_US&gl=US

You should be able to send the app directly to the device from that page.

I have not attempted what you are asking about.  It's a very good question and assumed that a Stream Assist device with location would be able to do this.  It may be best to post on the Stream Assist issues to see if someone there can confirm.

https://github.com/AlexxIT/StreamAssist/issues
